### PR TITLE
docs: added info on iam policy

### PR DESCRIPTION
**Pull request by dalsontws**
encountered issue with ZY when testing new distribution (CDC), reminder to create new policy for new distribution and assign permission to cli users

also reminder to use non-production key in development/stg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ An IAM account which has permissions to execute the `/auth` endpoint of the lamb
 
 Once the user is created, generate the `accessKeyId` and `secretAccessKey` for the production and staging endpoints.
 
+Disclaimer: Ensure that your IAM-CLI account has the required permission policy with the respective API ID specified in the ARN.
+This is important when adding a new distribution. E.g. ExecuteSupplyAllyPodStg
```
Comment (szenius) [inline README.md:12]: I don't think we should add the eg here since this is a public repo hahahaha

File: README.md [hunk 2]
```diff
@@ -44,8 +47,21 @@ Edit the last line of the script to determine:
 
 1. Prefix for the keys generated
 1. Number of keys to be generated
-1. If the key is for production use
+1. If the key is for production use (for stg, set as `false`)
 
 ```js
 makeKeys("TEST", 1, true);
```
Comment (szenius) [inline README.md:53]: Need to update this, we are using `${STAGE}-${IDENTIFIER}` for identity now. `makeKeys` also taking in an object now

File: README.md [hunk 3]
```diff
@@ -44,8 +47,21 @@ Edit the last line of the script to determine:
...
 ```
+
+## Output used to generate QR code
+
+In the resultant JSON output file, you will have the following:
+
+1. Endpoint
```
Comment (szenius) [inline README.md:60]: This is not in the JSON output file leh

File: README.md [hunk 4]
```diff
@@ -44,8 +47,21 @@ Edit the last line of the script to determine:
...
+2. Validity period (start and end in UTC format)
+3. Access Key(s)
+
+Using the fields generated above, proceed to fill the [voucher template](https://rationally.gdshive.io/) to generate your QR code.
+
+NOTE: If you are using a PR to generate a QR code, do take note to add the `STAGE` in the endpoint.
+i.e. `https://your.host.com/pr11`
```
Comment (szenius) [inline README.md:67]: Should this be under voucher-template instead?

File: README.md [hunk 5]
```diff
@@ -44,8 +47,21 @@ Edit the last line of the script to determine:
...
 ```js
 makeKeys("TEST", 1, true);
 ```
+
+## Output used to generate QR code
+
+In the resultant JSON output file, you will have the following:
+
+1. Endpoint
+2. Validity period (start and end in UTC format)
```
Comment (szenius) [inline README.md:61]: I think it is JS-UTC format, because it's the output from javascript's UTC() method. I can't find anything related to UTC format eh

File: README.md (hunk 1 above)
Comment (ilhaoni) [reply README.md:12]: +1 hahaha

File: README.md (hunk 3 above)
Comment (ilhaoni) [reply README.md:60]: It's in this other PR haha: https://github.com/rationally-app/voucher-generator/pull/3

File: README.md (hunk 4 above)
Comment (ilhaoni) [reply README.md:67]: Yeah... I dont think it should be here. I think we can update voucher-template to guide people on how to use, and in our docs repo we can explain how all the repos are used together. 

How does that sound? (:

File: README.md (hunk 1 above)
Comment (ilhaoni) [reply README.md:12]: Let's move this to docs also?

File: README.md [hunk 6]
```diff
@@ -18,14 +18,15 @@ cp .env.example .env
 
 and fill in the necessary credentials and the endpoint host and path as per configuration in API gateway.
 
-`STAGE` should be the seed stage that you want to generate vouchers for. For example, if you are trying to generate vouchers for `pod`, these variables will be expected from the `.env` file:
+`STAGE` should be the seed stage that you want to generate vouchers for. Do fill in your own identifier as well.
```
Comment (szenius) [inline README.md:21]: The instruction for identifier is actually in line 32!!

File: README.md [hunk 7]
```diff
@@ -18,14 +18,15 @@ cp .env.example .env
...
 ```
-STAGE=POD
+STAGE=YOUR_STAGE
+IDENTIFIER=YOUR_NAME
 STG_KEY_ID=<aws cli account key>
 STG_KEY_SECRET=<aws cli account secret>
-POD_STG_HOST=<pod_stg host>
-POD_STG_PATH=<pod_stg path>
+POD_STG_HOST=<stg host>
+POD_STG_PATH=<stg path>
```
Comment (szenius) [inline README.md:29]: Hmm why the change?

File: README.md [hunk 8]
```diff
@@ -40,12 +41,29 @@ PR11_STG_PATH=/pr11/auth
...
+
+## Output used to generate QR code
+
+In the resultant JSON output file, you will have the following:
+
+1. Endpoint
+2. Validity period (start and end in JS-UTC format)
+3. Access Key(s)
+
+Using the fields generated above, proceed to fill the [voucher template](https://rationally.gdshive.io/) to generate your QR code.
```
Comment (szenius) [inline README.md:69]: For mystique how 😬  Might want to ask them read voucher template readme here.

Also I thought the new endpoint for voucher template is supplyally.gdshive.io leh

File: README.md (hunk 6 above)
Comment (ilhaoni) [reply README.md:21]: Hmm, should we move line 32 up to be with this line? Would this make it clearer? Haha

File: README.md (hunk 8 above)
Comment (dalsontws) [reply README.md:69]: hmmmmmmm then should we just remove the endpoint? also im still using the rationally.gdshive.io LOL is still works! haha i'll change this.

Comment (szenius) [reply README.md]: Think need to update the url in voucher-template HAHA

File: README.md [hunk 9]
```diff
@@ -18,17 +18,19 @@ cp .env.example .env
 
 and fill in the necessary credentials and the endpoint host and path as per configuration in API gateway.
 
-`STAGE` should be the seed stage that you want to generate vouchers for. For example, if you are trying to generate vouchers for `pod`, these variables will be expected from the `.env` file:
-
 ```
-STAGE=POD
+IDENTIFIER=YOUR_NAME
```
Comment (ilhaoni) [inline README.md:22]: for this, can we write `YOUR_NAME` or `RECIPIENT`

cause sometimes we'll generate for external-team-a, so it'll be easier to track haha

File: README.md [hunk 10]
```diff
@@ -40,12 +42,29 @@ PR11_STG_PATH=/pr11/auth
...
+
+## Output used to generate QR code
+
+In the resultant JSON output file, you will have the following:
+
+1. Endpoint
+2. Validity period (start and end in JS-UTC format)
+3. Access Key(s)
+
+Using the fields generated above, proceed to fill the [voucher template](https://supplyally.gdshive.io/) to generate your QR code.
```
Comment (ilhaoni) [inline README.md:70]: Can we add a caveat here, to say these are for cases that do not use mystique HAHA

**Comment by szenius** [comment]
> Just a few more small changes~
> 
> I feel like lets create a card on trello, to see how we can automate voucher-generator and voucher-template? cause actually they can be quite interlinked, and things can get messy now. What do yall think? ((:

Yes plssssss

File: README.md (hunk 6 above)
Comment (dalsontws) [reply README.md]: okay done!!😀

File: README.md (hunk 7 above)
Comment (dalsontws) [reply README.md]: oops my bad wanted to change to a more generic naming, brain fart

File: README.md (hunk 8 above)
Comment (dalsontws) [reply README.md]: can! i think can just mention to use mystique URL for now....

File: README.md (hunk 10 above)
Comment (dalsontws) [reply README.md:70]: okay!!

File: README.md (hunk 9 above)
Comment (dalsontws) [reply README.md:22]: okay can!

**Comment by dalsontws** [comment]
> Just a few more small changes~
> 
> I feel like lets create a card on trello, to see how we can automate voucher-generator and voucher-template? cause actually they can be quite interlinked, and things can get messy now. What do yall think? ((:

agreed! maybe we can combine with this card? [trello](https://trello.com/c/xeHnp2we)

**Comment by ilhaoni** [comment]
> > Just a few more small changes~
> > I feel like lets create a card on trello, to see how we can automate voucher-generator and voucher-template? cause actually they can be quite interlinked, and things can get messy now. What do yall think? ((:
> 
> agreed! maybe we can combine with this card? [trello](https://trello.com/c/xeHnp2we)

yeahh, hmm, if its just a discussion i think its ok~ maybe we can add a implementation card instead? i have a feeling the implementation may take a bit of effort haha

File: README.md (hunk 7 above)
Comment (szenius) [reply README.md]: But still "stg host" leh, do you want to make it "your stage stg host" so they don't think it's the stg one
